src/fides/api/ops/api/v1/endpoints/connection_endpoints.py

```diff
@@ -129,7 +130,17 @@ def get_connections(
         )
 
     if connection_type:
-        query = query.filter(ConnectionConfig.connection_type.in_(connection_type))
+        saas_connection_types = []
+        for ct in connection_type:
```

Comment:
I think the connection types on the backend will always be lowercase? If so let's convert the incoming before searching to make it case insensitive.

```suggestion
        lower_connection_type = [x.lower() for x in connection_type]
        for ct in lower_connection_type:
```